exporter/awscloudwatchlogsexporter/config.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package awscloudwatchlogsexporter
+
+import (
+	"go.opentelemetry.io/collector/config/configmodels"
+	"go.opentelemetry.io/collector/exporter/exporterhelper"
+)
+
+// Config represent a configuration for the CloudWatch logs exporter.
+type Config struct {
+	configmodels.ExporterSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct.
+
+	exporterhelper.RetrySettings `mapstructure:"retry_on_failure"`
+
+	// LogGroupName is the name of CloudWatch log group which defines group of log streams
+	// that share the same retention, monitoring, and access control settings.
+	LogGroupName string `mapstructure:"log_group_name"`
+
+	// LogStreamName is the name of CloudWatch log stream which is a sequence of log events
+	// that share the same source.
+	LogStreamName string `mapstructure:"log_stream_name"`
+
+	// Region is the AWS region where the logs are sent to.
+	// Optional.
```

Comment:
What happens in the region is unspecified? Do we deliver to the region where the Collector is running?

Reply:
It uses the default region (set via aws-cli or the region the workload is running).